installscripts/jazz-terraform-unix-noinstances/scripts/modifyCodebase.sh

```diff
@@ -61,15 +61,15 @@ do
 			  "SERVICE_RUNTIME":{"S":"nodejs"},
 			  "SERVICE_STATUS":{"S":"active"},
 			  "TIMESTAMP":{"S":"'$timestamp'"},
+			  "SERVICE_TYPE":{"S":"lambda"},
```

Comment:
Please use separate PR's for respective stories. That way its easier for us to validate, track and manage our stories.

Reply:
going forward i will take care of this